### PR TITLE
Add PatrickXYS into reviewer list

### PR DESCRIPTION
**Pull request by PatrickXYS**
**Description of your changes:**
Hi all, I have continuously contributed to kubeflow/manifests repo and excited to continue working on it.

Thus, I want to add myself into reviewer list to help manifests grow fast.

The list of my PR on manifests:
https://github.com/kubeflow/manifests/pull/1410
https://github.com/kubeflow/manifests/pull/1330
https://github.com/kubeflow/manifests/pull/1155
https://github.com/kubeflow/manifests/pull/982
https://github.com/kubeflow/manifests/pull/953
https://github.com/kubeflow/manifests/pull/952

**Checklist:**
- [x] Unit tests have been rebuilt: 
    1. `cd manifests/tests`
    2. `make generate-changed-only`
    3. `make test`

### CODE REVIEW

File: tests/stacks/aws/test_data/expected/rbac.authorization.k8s.io_v1_role_centraldashboard.yaml [hunk 1]
```diff
@@ -24,5 +24,6 @@ rules:
   - ""
   resources:
   - secrets
+  - configmaps
```
Comment (terrytangyuan) [inline tests/stacks/aws/test_data/expected/rbac.authorization.k8s.io_v1_role_centraldashboard.yaml:27]: Is this change intended?

Comment (PatrickXYS) [reply tests/stacks/aws/test_data/expected/rbac.authorization.k8s.io_v1_role_centraldashboard.yaml:27]: Right, I did a search, should be this PR: https://github.com/kubeflow/manifests/pull/1394.

The PR's `run test` actually failed after merged. Developers may forget to re-generate tests before merge. Thus, it falls back to my side when I generate tests.
